browser/actors/FathomChild.jsm

```diff
@@ -30,4 +30,27 @@ class FathomChild extends JSWindowActorChild {
     // TODO: Something like Fathom.runRuleset(this.document);
     console.log(this.contentWindow.location.href);
   }
+
+  addCSSBorderAndLabel(type) {
+    let color = "";
+    switch (type) {
+      case "shopping":
+        color = "red";
+        break;
+      case "article":
+        color = "blue";
+        break;
+    }
+    this.document.body.style.border = `5px solid ${color}`;
+    const labelElement = this.document.createElement("SPAN");
+    labelElement.style.position = "absolute";
+    labelElement.style.padding = "10px";
+    labelElement.style.top = "0";
+    labelElement.style.left = "50%";
+    labelElement.style.transform = "translateX(-50%)";
+    labelElement.style.backgroundColor = "white";
```

Comment:
If we make the rect's bgcolor red and the text white, it will appear more like a tab hanging down from the border:

![Screen Shot 2019-11-20 at 12 11 20 ](https://user-images.githubusercontent.com/295816/69261149-104e0980-0b8f-11ea-93c8-75802ce69395.png)